interfaces/acados_matlab_octave/acados_ocp_opts.m

```diff
@@ -52,12 +52,27 @@
 			obj.opts_struct.codgen_model = 'true';
 			obj.opts_struct.param_scheme = 'multiple_shooting_unif_grid';
 			obj.opts_struct.param_scheme_N = 10;
+			% obj.opts_struct.param_scheme_shooting_nodes % only needed for nonuniform grid
 			obj.opts_struct.nlp_solver = 'sqp';
 			obj.opts_struct.nlp_solver_exact_hessian = 'false';
+			obj.opts_struct.nlp_solver_max_iter = 100;
+			obj.opts_struct.nlp_solver_tol_stat = 1e-6;
+			obj.opts_struct.nlp_solver_tol_eq = 1e-6;
+			obj.opts_struct.nlp_solver_tol_ineq = 1e-6;
+			obj.opts_struct.nlp_solver_tol_comp = 1e-6;
+			obj.opts_struct.nlp_solver_ext_qp_res = 0;
 			obj.opts_struct.qp_solver = 'partial_condensing_hpipm';
+			obj.opts_struct.qp_solver_iter_max = 50;
+			obj.opts_struct.qp_solver_cond_N = 5; % for partial condensing_hpipm
```

Comment:
this is unsafe, and will seg fault or the like if N<5

Reply:
if nothing is set, the C code will choose as default value N, i.e. no partial condensing